### PR TITLE
Increase Rooms Python Code Coverage

### PR DESCRIPTION
**Pull request by minnieliu**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-rooms/tests/test_rooms_client_async.py [hunk 1]
```diff
@@ -205,6 +215,24 @@ async def test_update_room_exceed_max_timerange_async(self):
...
 
+            with pytest.raises(HttpResponseError) as ex:
+                await self.rooms_client.update_room(room_id=create_response.id, valid_from=valid_from, valid_until=valid_until)
+
+                # delete created room
+                await self.rooms_client.delete_room(room_id=create_response.id)
+
+                assert str(ex.value.status_code) == "400"
+                assert ex.value.message is not None
+    @recorded_by_proxy_async
```
Comment (Mrayyan) [inline sdk/communication/azure-communication-rooms/tests/test_rooms_client_async.py:226]: Extreme nit: you can add a line break here

File: sdk/communication/azure-communication-rooms/tests/test_rooms_client.py [hunk 2]
```diff
@@ -90,9 +90,8 @@ def test_create_room_only_participants(self):
 
         self.verify_successful_room_response(response=response)
 
-    @pytest.mark.live_test_only
     @recorded_by_proxy
-    def test_create_room_validUntil_7Months_async(self):
+    def test_create_room_validUntil_7Months(self):
```
Comment (Mrayyan) [inline sdk/communication/azure-communication-rooms/tests/test_rooms_client.py:94]: could this be test_create_room_validUntil_seven_months or are numbers here fine?

**Review by Mrayyan — APPROVED** [review]

File: sdk/communication/azure-communication-rooms/tests/test_rooms_client.py (hunk 2 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-rooms/tests/test_rooms_client.py:94]: I can update this for consistency, thanks!

File: sdk/communication/azure-communication-rooms/tests/test_rooms_client_async.py (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-rooms/tests/test_rooms_client_async.py:226]: Sounds good, will add it!

File: sdk/communication/azure-communication-rooms/tests/test_rooms_client.py (hunk 2 above)
Comment (mikehang-msft) [reply sdk/communication/azure-communication-rooms/tests/test_rooms_client.py:94]: I was using exceed_180days

**Review by mikehang-msft — APPROVED** [review]

**Pull request merged**